matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/EventRelationsAggregationProcessor.kt

```diff
@@ -470,10 +463,13 @@ internal class EventRelationsAggregationProcessor @Inject constructor(
             return
         }
 
+        val pollOwnerId = getPollEvent(roomId, pollEventId)?.root?.senderId
+        val isPollOwner = pollOwnerId == sessionManager.getSessionComponent(sessionId)?.session()?.myUserId
```

Comment:
- You should not compare `pollOwnerId` with `myUserId` but rather with `event.senderId`.
- Also I would move those check at the beginning of the fun to avoid querying the database (`EventAnnotationsSummaryEntity` and `PollResponseAggregatedSummaryEntity`) for nothing.

Reply:
Omg, done. Thanks for catching.